release/_releases_json.sh

```diff
@@ -330,12 +354,14 @@ function _tag_recommended_product_versions {
 		then
 			lc_log INFO "Tagging ${latest_product_version_json_file} as recommended."
 
-			jq "map(
-					(. + {tags: [\"recommended\"]})
+			jq 'map(
+					.tags = ((.tags // []) + ["recommended"]
+					| unique
+					| sort)
 					| to_entries
 					| sort_by(.key)
 					| from_entries
-				)" "${latest_product_version_json_file}" > "${latest_product_version_json_file}.tmp" && mv "${latest_product_version_json_file}.tmp" "${latest_product_version_json_file}"
+				)' "${latest_product_version_json_file}" > "${latest_product_version_json_file}.tmp" && mv "${latest_product_version_json_file}.tmp" "${latest_product_version_json_file}"
```

Comment:
@GabrielFernandoS, for this kind changes (replace double quotes by single quotes), try to do atomic commits, e.g. a commit, theoretically, should do a single action.